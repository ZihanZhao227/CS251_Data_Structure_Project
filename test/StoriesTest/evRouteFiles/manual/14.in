2
0 0 0 CITY StartEnd
1 50 0 CHARGING_STATION CS1
