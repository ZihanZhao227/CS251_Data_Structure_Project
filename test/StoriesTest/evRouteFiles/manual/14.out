0 0 100.000 0.000
0
